REFERENCE

```diff
@@ -0,0 +1,21 @@
+****************************************************************************************************
+All OPENAIRINTERFACE References are listed below and are covered under Apache 2.0 License.
```

Comment:
Would be good if all the files we add new documentation related files with markdown format.

Reply:
# LICENSE INFORMATION
>This page should capture all third party software used in the project **OMEC** and list the license information of those components. Even if full information about the license is not available its always good to list the third party library, component, source code here.
>
## OPENAIRINTERFACE License  

- openinterface is covered under APACHE 2.0 license
-- https://github.com/OPENAIRINTERFACE/openair-cn/tree/master/src/hss_rel14/hsssec : c3po/hsssec
-- https://github.com/OPENAIRINTERFACE/openair-cn/tree/master/src/hss_rel14/util   : c3po/util

## DPDK License
- DPDK Reference is covered under Apache 2.0 License.

## SSL License 
- Reference to SSL is covered under Apache 2.0 License. 
-- https://tls.mbed.org/ : c3po/sgxcdr/dealer/Enclave

## Eclipse License
- Reference to Eclipse SGX is covered under EPL 1.0 License.
-- https://01.org/intel-softwareguard-extensions : c3po/sgxcdr/kms/App
-- https://01.org/intel-softwareguard-extensions : c3po/sgxcdr/dealer/App

Reply:
Use this file in raw format with .md extension. Also update content if something is wrong.